### PR TITLE
Feat/debug things

### PR DESCRIPTION
**Pull request by gwillz**
When inspecting a page we often tack on a `<!-- path/to/template -->` comment to help us find our way around. So this does it automatically (within non-production) for each view render.

This PR also has a fatal error handler. It's not something we see a lot of, but it makes prettier error pages of compiling or input errors. 

I probably want this in the v3.3 release but I'm twitchy about it for some reason. Any thoughts?

### CODE REVIEW

**Comment by gwillz** [comment]
bump?

**Review by aitken85 — APPROVED** [review]

File: src/sprout/core/Kohana.php [hunk 1]
```diff
@@ -692,6 +698,25 @@ public static function errorHandler($errno, $errmsg, $file, $line)
         throw new ErrorException($errmsg, 0, $errno, $file, $line);
     }
 
+
+    public static function handleFatalErrors()
```
Comment (jamiemonksuk) [inline src/sprout/core/Kohana.php:702]: Could be pedantic about a docblock here, but the name is self evident :D

**Review by jamiemonksuk — APPROVED** [review]

File: src/sprout/core/Kohana.php (hunk 1 above)
Comment (gwillz) [reply src/sprout/core/Kohana.php:702]: No, please!

I'll add one now.

**Comment by gwillz** [comment]
before
image

after
image

**Pull request merged**